HST/run_validation.py

```
@@ -38,11 +41,18 @@ def run_validation(proposal_id, logger=None):
     create_manifest_files(proposal_id, logger)
 
     bundle_dir = get_program_dir_path(proposal_id, None, root_dir='bundles')
-    run(["./validate-pdart", bundle_dir, bundle_dir, bundle_dir, str(proposal_id)])
+    logger.info(f'Run validation-pdart for proposal id: {proposal_id} on {bundle_dir}')
+    try:
+        run([VALIDATE_PDART, bundle_dir, bundle_dir, bundle_dir, str(proposal_id)],
+            cwd=HST_DIR, check=True)
+    except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)